gpt4all-backend/replit_impl.h

```diff
@@ -0,0 +1,40 @@
+#ifndef REPLIT_H_I_KNOW_WHAT_I_AM_DOING_WHEN_INCLUDING_THIS_FILE
+#error This file is NOT meant to be included outside of replit.cpp. Doing so is DANGEROUS. Be sure to know what you are doing before proceeding to #define REPLIT_H_I_KNOW_WHAT_I_AM_DOING_WHEN_INCLUDING_THIS_FILE
+#endif
+#ifndef REPLIT_H
+#define REPLIT_H
+
+#include <string>
+#include <functional>
+#include <vector>
+#include "llmodel.h"
+
+#define GGML_QNT_VERSION_FACTOR 1000 // do not change this
+
+class ReplitPrivate;
```

Comment:
this is a `struct` in `replit.cpp` so it should be a `struct` here too.

```diff
diff --git a/gpt4all-backend/replit_impl.h b/gpt4all-backend/replit_impl.h
index afbd84e..28b7493 100644
--- a/gpt4all-backend/replit_impl.h
+++ b/gpt4all-backend/replit_impl.h
@@ -11,7 +11,7 @@
 
 #define GGML_QNT_VERSION_FACTOR 1000 // do not change this
 
-class ReplitPrivate;
+struct ReplitPrivate;
 class Replit : public LLModel {
 public:
     Replit();
```

Reply:
fixed!